pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -150,7 +150,7 @@ func newCertRotationController(
 			Name:      "kube-apiserver-aggregator-client-ca",
```

Comment:
is this the CA that is added to `--requestheader-client-ca-file` ?

Reply:
Yes (that's the entire CA btw)